packages/aws-cdk-lib/aws-synthetics/lib/runtime.ts

```diff
@@ -190,7 +190,7 @@ export class Runtime {
    * - **Ephemeral storage monitoring**: This runtime adds ephemeral storage monitoring in customer accounts.
    * - **Bug fixes**
    *
-   * @see https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch_Synthetics_Library_nodejs_puppeteer.html#CloudWatch_Synthetics_runtimeversion-nodejs-puppeteer-6.1
+   * @see https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch_Synthetics_Library_nodejs_puppeteer.html#CloudWatch_Synthetics_runtimeversion-nodejs-puppeteer-6.2
```

Comment:
nit: this change, although necessary, may get hidden in history because it is being done with addition of Node.js puppeteer 8 runtime, hence, is it possible to take this separately (if not much effort required)?

Reply:
@ChakshuGupta13 
Thanks.
I've separated a PR #30853 to fix wrong link.